Add missing arguments to boot_SDL(); this fixes a crash or failure in
Perl_xs_handshake() on some arches.  This patch does pass the CV of
the wrong function, but the handshake accepts it.

Index: src/SDL.xs
--- src/SDL.xs.orig
+++ src/SDL.xs
@@ -69,7 +69,7 @@ static int sdl_perl_use_smpeg_audio = 0;
 #define HAVE_TLS_CONTEXT
 #endif
 
-#include "defines.h"
+#include "src/defines.h"
 
 Uint32 
 sdl_perl_timer_callback ( Uint32 interval, void* param )
@@ -189,13 +189,13 @@ sdl_perl_atexit (void)
 #endif
 }
 
-void boot_SDL();
+void boot_SDL(pTHX_ CV *);
 void boot_SDL__OpenGL();
 
 XS(boot_SDL_perl)
 {
 	GET_TLS_CONTEXT
-	boot_SDL();
+	boot_SDL(aTHX_ cv);
 }
 
 MODULE = SDL_perl	PACKAGE = SDL
@@ -3911,7 +3911,7 @@ GFXFilledpieColor ( dst, x, y, rad, start, end, color 
     Sint16 end;
     Uint32 color;
 CODE:
-     RETVAL = filledpieColor( dst, x, y, rad, start, end, color );
+     RETVAL = filledPieColor( dst, x, y, rad, start, end, color );
 OUTPUT:
      RETVAL
 
@@ -3928,7 +3928,7 @@ GFXFilledpieRGBA ( dst, x, y, rad, start, end, r, g, b
     Uint8 b;
     Uint8 a;
 CODE:
-     RETVAL = filledpieRGBA( dst, x, y, rad, start, end, r, g, b, a );
+     RETVAL = filledPieRGBA( dst, x, y, rad, start, end, r, g, b, a );
 OUTPUT:
      RETVAL
 
